packages/viewer/scripts/download_duckdb_extensions.py

```diff
@@ -5,7 +5,7 @@
 
 remote_repo = "https://extensions.duckdb.org"
 
-versions = ["v1.1.1"]
+versions = ["v1.3.2"]
```

Comment:
Is this the duckdb version? We might want to bump to 1.3.4 because of https://github.com/duckdb/duckdb-node/security/advisories/GHSA-w62p-hx95-gf2c

Reply:
Ah, I see. No, I don't know.